docs/docsite/rst/user_guide/playbooks_intro.rst

```diff
@@ -376,7 +376,7 @@ Ansible prefers listing modules like this::
         src: templates/foo.j2
         dest: /etc/foo.conf
 
-Early versions of Ansible used the following format, which still works::
+Early versions of Ansible used the following format, which still works (but is not recommended)::
```

Comment:
i actually recommend it for readability in some cases

Reply:
Darn! I just can't win! :-P